docs/src/configuration/app/firewall.md

```diff
@@ -0,0 +1,81 @@
+# Outbound firewall
```

Comment:
```suggestion
---
title: "Outbound firewall"
weight: 15
---
```

Reply:
This will add it to the sidebar.

Reply:
I'm not sure if we want this to show up last in that section, but  that's what this weight will do.